aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -332,7 +332,12 @@ struct Vec256 {
     return map(std::expm1);
   }
   Vec256<T> frac() const {
-    return *this - this->trunc();
+    return map([](const T& x) -> T {
```

Comment:
Can this use std::modf instead?

Reply:
Since this part generated `c10::Half` Vec256, we can't use std::modf as it is not implemented for `c10::Half`